pkg/cmd/chat/channel/channel.go

```diff
@@ -0,0 +1,230 @@
+package channel
+
+import (
+	"encoding/json"
+	"errors"
+	"text/tabwriter"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/cheynewallace/tabby"
+	"github.com/spf13/cobra"
+)
+
+func NewChannelCmds() []*cobra.Command {
+	return []*cobra.Command{
+		getChannelCmd(),
+		createChannelCmd(),
+		deleteChannelCmd(),
+		updateChannelCmd(),
+		listChannelsCmd()}
```

Comment:
same here, package is already explicit

```suggestion
		getCmd(),
		createCmd(),
		deleteCmd(),
		updateCmd(),
		listCmd()}
```